_posts/0000-01-12-aniciu.md

```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![chill-out-lemur](https://cloud.githubusercontent.com/assets/16547949/25400761/9c64ee82-29c1-11e7-8ad3-6886f97634b9.jpg)
 {: .center}
 
-Caption here
+Stop dude, and don't talk so fast
```

Comment:
Hopefully this isn't about me 😁 

Maybe use `**bold**` or `_italics_` in your markdown file 👍

Reply:
@beardofedu Of course is not about you. You rock! 👍